### PR TITLE
Saurabsa/add reference environment

### PR DESCRIPTION
**Pull request by saurabsa**

### CODE REVIEW

File: src/command_modules/azure-cli-project/azure/cli/command_modules/project/references.py [hunk 1]
```diff
@@ -133,15 +137,17 @@ def _create_custom_reference(service_name, reference_name, key_value_pairs):
         env_variables.append(key)
         literals.append('--from-literal={}={}'.format(key, value))
 
-    command = 'kubectl create secret generic {} {}'.format(
-        secret_name, ' '.join(literals))
+    command = 'kubectl create secret generic {} {} -n {}'.format(
+        secret_name, ' '.join(literals), namespace_name)
+    print(command)
```
Comment (PeterJausovec) [inline src/command_modules/azure-cli-project/azure/cli/command_modules/project/references.py]: remove the print statement

File: src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py [hunk 2]
```diff
@@ -1285,6 +1304,14 @@ def _run_innerloop_command(*args):
         raise CLIError(error)
 
 
+def _get_innerloop_command_output(*args):
+    """
+    Calls InnerLoop client to set up a tenx project for a multi-container Docker application.
```
Comment (PeterJausovec) [inline src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py]: the comment doesn't seem correct

File: src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py [hunk 3]
```diff
@@ -1169,6 +1173,14 @@ def _initialize_workspace(
     logger.info('\nCluster configured successfully.')
 
 
+def _get_current_environment_output():
```
Comment (PeterJausovec) [inline src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py]: if there's no need to parse anything from the output, let's just call the method ```_get_current_environment```

File: src/command_modules/azure-cli-project/azure/cli/command_modules/project/references.py (hunk 1 above)
Comment (saurabsa) [inline src/command_modules/azure-cli-project/azure/cli/command_modules/project/references.py]: Oh Yes missed it. I would have been a bad surgeon.

File: src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py [hunk 4]
```diff
@@ -1169,6 +1173,14 @@ def _initialize_workspace(
     logger.info('\nCluster configured successfully.')
 
 
+def _get_current_environment():
+    """
+    Calls tenx run command on the current directory.
```
Comment (PeterJausovec) [inline src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py]: extra comment?

File: src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py [hunk 5]
```diff
@@ -1260,19 +1272,26 @@ def _service_remove_reference(reference_name):
...
+    if not os.path.isfile(cmd):
+        cmd = 'dotnet ' + os.path.join(file_path, 'tenx.dll')
+
+    cmd = cmd + ' ' + ' '.join(args)
+    return cmd
+
 
 def _run_innerloop_command(*args):
     """
     Calls InnerLoop client to set up a tenx project for a multi-container Docker application.
```
Comment (PeterJausovec) [inline src/command_modules/azure-cli-project/azure/cli/command_modules/project/custom.py]: wrong comment here as well? Shouldn't this be - Runs an innerloop command or something like that?

**Pull request merged**